pkg/config/reload.go

```diff
@@ -228,3 +231,32 @@ func (c *Config) ReloadRdtConfig(newConfig *Config) error {
 	}
 	return nil
 }
+
+// ReloadRuntimes reloads the runtimes configuration if changed
+func (c Config) ReloadRuntimes(newConfig *Config) error {
+	var newRuntimes []string
+	for runtime := range newConfig.Runtimes {
+		if _, ok := c.Runtimes[runtime]; !ok {
+			continue
+		}
+		newRuntimes = append(newRuntimes, runtime)
+	}
+
+	for _, runtime := range newRuntimes {
+		c.Runtimes[runtime] = newConfig.Runtimes[runtime]
+	}
```

Comment:
couldn't we just compress these lines and do 
`		c.Runtimes[runtime] = newConfig.Runtimes[runtime]` if it wasn't found?